src/wp-includes/class-wp-duotone.php

```diff
@@ -1056,22 +1068,29 @@ public static function set_global_style_block_names() {
 	 *
 	 * @since 6.3.0
 	 *
-	 * @param  string $block_content Rendered block content.
-	 * @param  array  $block         Block object.
-	 * @return string                Filtered block content.
+	 * @param  string   $block_content Rendered block content.
+	 * @param  array    $block         Block object.
+	 * @param  WP_Block $wp_block      The block instance.
+	 *
+	 * @return string Filtered block content.
```

Comment:
```suggestion
	 * @param string   $block_content Rendered block content.
	 * @param array    $block         Block object.
	 * @param WP_Block $wp_block      The block instance.
	 * @return string Filtered block content.
```

As per the coding standard for [function formatting](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods), update the code accordingly.